README.md

```diff
@@ -1,84 +1,93 @@
+| `disable_detect` | bool | Optional| If `false`, any faces detected but not verified as matching a label will be labeled as "face".<br> Default: `false` | - |
+| `disable_verify` | bool | Optional | If `false` and `face_labels` are set, will attempt to verify any faces detected. If you only want verified faces, set this and `disable_detect` to `true`.<br> Default: `false` | - |
 
-*object*
 
-If configured, expects an object map of key:label, value:path to use in matching against reference face images.
-For example:
+### Example configuration 
 
-``` json
+```json
 {
-  "matt": "/path/to/matt.jpg",
-  "suzy": "/path/to/suzy_photo.jpg"
+  "detection_framework": "ssd",
+  "recognition_model": "ArcFace",
+  "face_labels": {
+    "matt": "/path/to/matt.jpg",
+    "suzy": "/path/to/suzy_photo.jpg"
+  },
+  "verify_threshold": 0.8,
+  "disable_detect": false,
+  "disable_verify": false
 }
 ```
 
-If the input image from get_detections() or get_detections_from_camera() verifies as a match of one of the images specified in the *face_labels* paths, the associated label will be returned.
-
-### verify_threshold
-
-*number(default:.8)*
-
-If [disable_verify](#disable_verify) is false and [face_labels](#face_labels) are set, if the verification confidence does not match or exceed this threshold, it will return as a normal detected "face".
-
-### disable_detect
-
-*boolean(default:false)*
+## API
 
-If [disable_verify](#disable_verify) is false and [face_labels](#face_labels) are set, if disable_detect is false, any faces detected but not verified as matching a label will be labeled as *face*.
+The `facial-detector` resource provides the following methods from [Viam's vision API](https://python.viam.dev/autoapi/viam/services/vision/client/index.html).
 
-### disable_verify
+| Method   | Parameters | Description |
+|--------|-------|----------|
+| [`get_detections`](https://python.viam.dev/autoapi/viam/services/vision/client/index.html#viam.services.vision.client.VisionClient.get_detections) | `image` | Get a list of detections in the given image using the specified detector.|
+ | [`get_detections_from_camera`](https://python.viam.dev/autoapi/viam/services/vision/client/index.html#viam.services.vision.client.VisionClient.get_classifications_from_camera) | `camera_name=string` | Get a list of classifications in the next image given a camera and a classifier.|
 
-*boolean(default:false)*
+### Next steps
 
-If false and [face_labels](#face_labels) are set, will attempt to verify any faces detected.
-If you only want verified faces to be labeled, set this to false and [disable_detect](#disable_detect) to true.
+[Create a Facial Verification System](https://docs.viam.com/tutorials/projects/verification-system/#configure-a-verification-system) using the facial detector you configured.
```

Comment:
Is there anything else we can suggest to people just to check that the module is working? I think we usually suggest something they can confirm on the control tab.
And then I'd suggest making this sound like an option the person has. They can use this module for other purposes after all:

```suggestion
Check out the [Create a Facial Verification System](https://docs.viam.com/tutorials/projects/verification-system/#configure-a-verification-system) tutorial that uses this module.
```